include/orb/JacobianHelpers/jacobian_autocoded.h

```diff
@@ -0,0 +1,171 @@
+/*
+MIT License
+
+Copyright (c) 2020 Pathfinder for Autonomous Navigation (PAN)
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+*/
+
+/**
+ * \file jacobian_autocoded.hpp
+ * \author Nathan Zimmerberg
+ * \date 10 APR 2020
+ * \brief Helper function to get the jacobian of in Orbit::shortupdate().
+ * \details  Partially auto code from sympy in jacobian_autocoder.py
+ */
+
+#pragma once
```

Comment:
I think we should still stay away from GNU extensions even if they're pretty widely supported. If we could stick to `#ifdef` guards even if it's just for the sake of consistency that'd be great.

Reply:
pragma once is used at least 149 times in FlightSoftware. If there ever is a problem its not too hard to fix.

Reply:
Pretty much every use of pragma once in fsw comes from the Arduino Json library and not our own code, but I'm not gonna beat a dead horse.